packages/protocol/contracts/layer1/tiers/TierProviderBase.sol

```diff
@@ -14,7 +14,7 @@ abstract contract TierProviderBase is ITierProvider {
     /// asynchronous, this grace period allows provers to defer submissions until gas
     /// prices become more favorable, potentially reducing transaction costs.
     uint16 public constant GRACE_PERIOD = 240; // minutes
-    uint96 public constant BOND_UNIT = 75 ether; // TAIKO tokens
+    uint96 public constant BOND_UNIT = 0.04 ether;
```

Comment:
double checking numbers here too

Reply:
Same interpretation as the previous value of 0.07 eth. Based on Taiko price.